projects/models.py

```diff
@@ -159,3 +178,48 @@ def add_components_for_project(sender, instance, **kwargs):
                 instance.components.add(aws_component)
         except ObjectDoesNotExist as e:
             logger.warning(f"Inherited components not found: {e}")
+
+
+class Control(models.Model):
+    class Status(models.IntegerChoices):
+        NOT_STARTED = 1, "Not started"
+        INCOMPLETE = 2, "Incomplete"
+        COMPLETE = 3, "Complete"
+
+    class Responsibility(models.IntegerChoices):
+        ALLOCATED = 1, "Allocated"
+        SHARED = 2, "Shared"
+        INHERITED = 3, "Inherited"
+
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
```

Comment:
Yes, the Controls in that table are Project specific, so I think that it would be best to delete them otherwise we'll end up with a bunch of orphaned data.

Reply:
From our conversation about alternate workflow, we may need to reevaluate this.